src/__test__/__snapshots__/Home.test.js.snap

```diff
@@ -0,0 +1,175 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Calculator Renders 1`] = `
```

Comment:
- Good work implementing your test suites. However, when I'm running the test, this is what I'm having:

![Screenshot from 2023-05-04 12-40-35](https://user-images.githubusercontent.com/97834160/236182656-1bb1bc25-1bcd-44c5-b9d3-e4f1a92d2947.png)

It seems like the snapshots are now considered obsolete, meaning that the output of the test has changed since the last time the snapshot was generated.
I think it would be better to update your snapshots in order to match the updated output